static/templates/communication-costs.hbs

```diff
@@ -0,0 +1,85 @@
+<div class="panel__row">
+  <h4 class="panel__title">Disbursement information</h4>
+  <table>
+    <tr>
+      <td class="panel__term">Amount</td>
+      <td class="panel__data">{{ currency disbursement_amount }}</td>
+    </tr>
+    <tr>
+      <td class="panel__term">Report year</td>
+      <td class="panel__data">{{ report_year }}</td>
+    </tr>
+    <tr>
+      <td class="panel__term">Transaction date</td>
+      <td class="panel__data">{{ datetime transaction_date pretty=true }}</td>
+    </tr>
+    <tr>
+      <td class="panel__term">Communication class</td>
```

Comment:
I am not sure we need to display communication class. Right now, we are showing  codes, and the options are M, E or S which I think is membership, executive personnel or stockholders. If we want to display this information, we will want to decode it. Also, it is blank in a third of the data.